tests/nlp/checks/model_evaluation/single_dataset_performance_nlp_test.py

```diff
@@ -105,3 +105,46 @@ def test_run_with_scorer_multilabel_class_names(text_multilabel_classification_d
     # Assert
     assert_that(result.value.values[0][-1], close_to(1.0, 0.001))
     assert_that(result.value.values[0][0], equal_to('a'))
+
+
+def test_run_with_scorer_token(text_token_classification_dataset_mock):
```

Comment:
Also check different scenarios of wrong input format,
order of stuff in tuples in predictions
order of stuff in tuples in labels
labels and prediction that disagree - in labels str, in locations

Reply:
@noamzbr

Reply:
labels and prediction that disagree in str and locations is tested in test_run_with_scorer_token_per_class, can be seen in test_run_with_scorer_token_per_class.
Will implement the rest